packages/eas-cli/src/utils/profiles.ts

```diff
@@ -70,3 +76,25 @@ async function readProfileAsync<T extends ProfileType>({
     )) as EasProfile<T>;
   }
 }
+
+function maybePrintBuildProfileDeprecationWarnings(
+  buildProfile: BuildProfile<Platform>,
+  profileName?: string
+): void {
+  const deprecationWarnings = EasJsonUtils.getBuildProfileDepreactionWarnings(
+    buildProfile,
+    profileName
+  );
+  if (deprecationWarnings.length > 0) {
```

Comment:
This saves one level of indentation. 
```suggestion
  if (deprecationWarnings.length === 0) {
    return;
  }
```